src/components/header/header.js

```diff
@@ -18,17 +27,25 @@ const Header = () => {
                 <HeaderTitle>Awesome Container Company</HeaderTitle>
             </HeaderLeft>
             <HeaderRight>
-                {headerLinks.map((link) => {
-                    return (
-                        <HeaderLink
-                            href={link.id}
-                            key={headerLinks.indexOf(link)}>
-                            {link.name}
-                        </HeaderLink>
-                    )
-                })}
+                <Hamburger src={menuOpen ? menuExit : hamburger}
+                    onClick={toggleMenu} />
+                <HeaderNav>
+                    {headerLinks.map((link) => {
+                        return (
+                            <HeaderLink
+                                href={link.id}
+                                key={headerLinks.indexOf(link)}>
+                                {link.name}
+                            </HeaderLink>
+                        )
+                    })}
+                </HeaderNav>
                 <HeaderWork>Work with us</HeaderWork>
             </HeaderRight>
+            {menuOpen &&
```

Comment:
If you open the menu then increase the window size to something larger like 1024, the menu will remain open. There's a cool library called React-sizeme that can help with this. It makes components aware of their width, so you can do some logic like "if width is greater than X, close the menu". https://www.npmjs.com/package/react-sizeme

Reply:
That is a cool library.  I tried it out, but an issue I had was when I sized up then sized back down, the hamburger was still on X as opposed to the three lines (because it never got clicked, the menu just disappeared).  In an attempt to solve that problem I realized the whole thing would be simpler with the addition of a window event listener that pays attention to the screen width.